package.json

```diff
@@ -23,15 +23,15 @@
   },
   "dependencies": {
     "accounting-js": "^1.1.1",
-    "authorize-net": "ongoworks/node-authorize-net",
+    "authorize-net": "github:ongoworks/node-authorize-net",
```

Comment:
Why this change? From NPM docs "As of version 1.1.65, you can refer to GitHub urls as just "foo": "user/foo-project". Just as with git URLs, a commit-ish suffix can be included. For example:"